src/components/MainLayout.tsx

```diff
@@ -1,19 +1,28 @@
 import React from 'react';
+import { useTranslation } from 'react-i18next';
 import { Outlet } from 'react-router-dom';
+import useUserRole, { UserRole } from '../api/useUserRole';
 import Divider from './common/Divider';
 import Header from './Header';
 import styles from './MainLayout.module.scss';
 
-const MainLayout = (): React.ReactElement => (
-  <div className={styles['main-layout']}>
-    <Header />
-    <Divider />
-    <div className={styles['content-container']}>
-      <div className={styles.gutter}>
-        <Outlet />
+const MainLayout = (): React.ReactElement => {
+  const userRole = useUserRole();
+  const { t } = useTranslation();
+  return (
+    <div className={styles['main-layout']}>
+      <Header />
+      <Divider />
+      <div className={styles['content-container']}>
+        <div className={styles.gutter}>
+          {userRole > UserRole.NON_AD_GROUPS && <Outlet />}
+          {userRole === UserRole.NON_AD_GROUPS && (
+            <div>{t(`message.unauthorized`)}</div>
+          )}
```

Comment:
Ah, this is what's causing the login view to not render. If you're not logged in, it renders nothing.

Reply:
Fixed it.